docs/commands.md

```diff
@@ -58,6 +65,7 @@ Possible values are:
 * `CTRL_MODE_POSITION_CONTROL`
 * `CTRL_MODE_VELOCITY_CONTROL`
 * `CTRL_MODE_CURRENT_CONTROL`
+* `CTRL_MODE_TRAJECTORY_CONTROL`
```

Comment:
There is no such control mode

Reply:
Absolutely agree, that came from when I was reworking the documentation on `master`. Fixed with `573ad8f`